File: src/commands/avatar.js
module.exports = {
    name: "avatar",
    alias: [],
    botPermissions: [],
    run: async (client, message, args) => {
        const package = require('../../package.json')
        //Obtenemos el usuario
        let user = message.mentions.users.first()??client.users.cache.get(args[0])??message.author;
        //Se crea un embed
        const embed = new MessageEmbed()
            .setImage(user.displayAvatarURL({
                dynamic: true, 
                size: 512
            })) // El avatar
            .setTitle(`Avatar de ${user.username}`) // El titulo
            .setFooter(`${client.user.username} Bot ${package.version}`, client.user.avatarURL())
            .setFooter(`${client.user.username} ${package.version}`, client.user.avatarURL())
            .setTimestamp();
        message.inlineReply(embed);
    }
}

File: py/commands/avatar.py
import tools
from tools.utils import commands
import requests
import aiohttp

@tools.bot.command()
async def avatar (ctx, member : tools.discord.Member = None):
    translations = commands.get_config(ctx, "avatar")

    if member == None: 
        mem = ctx.author
    else: mem = member

    async with aiohttp.ClientSession(headers={'Content-Type': 'application/json', 'Authorization': f"Bot {tools.env.TOKEN}"}) as session:
        response = await session.get(f"https://discord.com/api/v9/guilds/{ctx.guild.id}/members/{mem.id}")
        data = await response.json()
        if data['avatar'] == None:
            avatar = mem.avatar_url
        else: avatar = f"https://cdn.discordapp.com/guilds/{ctx.guild.id}/users/{mem.id}/avatars/{data['avatar']}.webp?size=2048"
        embed = tools.discord.Embed(colour = mem.color, timestamp = tools.datetime.utcnow())
        embed.set_author(name = translations["embed"]["author"].format(mem), url = avatar)
        embed.set_image(url = avatar)
        embed.set_footer(icon_url = ctx.author.avatar_url, text = translations["embed"]["footer"].format(ctx.author.name))

        await ctx.send(embed = embed)
